src/app/test/db/mongo/pruner_test.py

```diff
@@ -36,6 +72,14 @@ def test_prune_something(self, pruner, collection_mock):
         pruner.run()
         assert collection_mock.objects.return_value.no_cache.return_value.delete.called
 
+    def test_run_deletes_outstanding_tasks(self, pruner, in_progress, created):
```

Comment:
This test name is misleading, since the desired behavior is that it cancels rather than deletes.  Also, they are "requests", rather than "tasks".